tickets/src/main/java/no/unit/nva/publication/ticket/update/UpdateTicketViewStatusHandler.java

```diff
@@ -71,7 +71,18 @@ private TicketEntry markTicketForElevatedUser(UpdateViewStatusRequest input, Req
         markTicketForElevatedUser(input, oldTicket);
         return oldTicket;
     }
-    
+
+    private void markTicketForElevatedUser(UpdateViewStatusRequest input, TicketEntry ticket) {
+
+        if (ViewStatus.READ.equals(input.getViewStatus())) {
+            ticket.markReadForCurators().persistUpdate(ticketService);
+        } else if (ViewStatus.UNREAD.equals(input.getViewStatus())) {
+            ticket.markUnreadForCurators().persistUpdate(ticketService);
+        } else {
+            throw new UnsupportedOperationException("Unknown ViewedStatus");
+        }
+    }
+
```

Comment:
This is a reformat, right?

Reply:
No, I moved the method to keep overloaded methods together. Codestyle warning.